--- UZIP.js	2025-08-20 15:11:42.289374851 -0400
+++ UZIP.mjs	2025-08-20 15:11:42.289374851 -0400
@@ -1,7 +1,7 @@
 
 
 
-var UZIP = {};
+const UZIP = {};
 
 (function(){
 	
@@ -312,7 +312,7 @@
 			*/
 			out[name] = buf;
 		}
-		else if(cmpr==14 && window["LZMA"]) {
+		else if(cmpr==14 && globalThis["LZMA"]) {
 			var vsn = rUs(file,0);
 			var siz = rUs(file,2);  if(siz!=5) throw "unknown LZMA header";
 			
@@ -328,7 +328,7 @@
 			var time = Date.now();
 			var buf = out[name] = new Uint8Array(usize);
 						
-			var dec = new window["LZMA"]["Decoder"]();
+			var dec = new globalThis["LZMA"]["Decoder"]();
 			dec["setProperties"]({ "dsz":dictSize, "lc":lc, "lp":lp, "pb":pb });
 			dec["decodeBody"](new Uint8Array(data.buffer,o+9),buf, usize);
 			
@@ -1017,3 +1017,7 @@
 	
 })();
 
+export {
+    UZIP,
+    UZIP as default
+};
